docs/editors.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: Editors
+description: Learn how to use Babel in your editor of choice
+permalink: /docs/editors
+---
+
+## Syntax Highlighting
+
+These days, many popular editors support ES2015+ syntax highlighting
```

Comment:
Would it be interesting to have a screen of syntax highlighting somewhere on here?